README.md

```diff
@@ -234,16 +234,16 @@ It's possible that your topic and system are entirely ok with losing some messag
     - `KAFKA_CLIENT_CERT_KEY`
     - `KAFKA_TRUSTED_CERT`
 
-2. Start `Kaffe.Producer` with your application
-
-      ```elixir
-      worker(Kaffe.Producer, [])
-      ```
-
 ### Producing to Kafka
 
 Currently only synchronous message production is supported.
 
+You'll need to start the `Kaffe.Producer`, this may be best suited within your applications callback:
+
+```elixir
+Kaffe.Producer.start_producer_client()
```

Comment:
My apps run in containers, configuration is produced through env/secret files and I use `:weave` to inject that config. So Kaffe doesn't start it for me, though I forgot that was why. I will remove this line